src/main/java/io/jenkins/plugins/orka/helpers/CapacityHandler.java

```diff
@@ -102,6 +103,9 @@ public void removeFailedPlannedInstance() {
         try {
             instanceLock.lock();
             this.plannedInstances--;
```

Comment:
Do you need this line ? Shouldn't we delete it ? If we decrease the `plannedInstances` here and on the line below, we'll decrease them 2 times 👀